app/bundles/CoreBundle/Doctrine/Type/ArrayType.php

```diff
@@ -39,6 +39,10 @@ function (&$entry) {
 
         $serialized = serialize($value);
 
+        if(strpos($serialized, chr(0)) !== false) {
```

Comment:
Replacing null bytes with nothing is wrong. You save A into database and get B from it. In this case properties will have diferrent name after deserialization back to objects. Replacing null bytes can lead into even stranger errors later on. Better to crash on writing then crash on all read operations.

I'm testing the code with throwing and exception and iplookup patch and I'm getting no more errors. This means there are probably no more objects with private properties in mautic source.

If it is needed to save objects with private properties just convert database column types to blob and all will be OK. :)